qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visitTypes.feature

```diff
@@ -1 +1,33 @@
-Feature: Visit types Management
\ No newline at end of file
+Feature: Visit types Management
+
+  Background:
+    Given a user clicks on the system administration link from the home page
+    And the system loads the administration page and User clicks on the manage Visit Types link
+
+  @selenium
+  @visitType
+  Scenario: Adding a Visit Type
+     When a user clicks on the add visit type link on the manage visit type page
+     And a user fills the add visit type form
+     And a user saves visit type
+     Then a visit type is saved successfully
+
+  @selenium
+  @visitType
+  Scenario: Editing a Visit Type Test
+     When a user clicks on the already saved visit type link
```

Comment:
what happens when the user click on already saved visit type link?

Reply:
Any suggestions @kdaud

Reply:
> Any suggestions @kdaud


have you taken a look at the existing feature files user stories?